samtranslator/model/eventsources/push.py

```diff
@@ -829,6 +831,8 @@ def _add_swagger_integration(  # type: ignore[no-untyped-def] # noqa: PLR0912, P
             self.add_auth_to_swagger(
                 self.Auth, api, api_id, self.relative_id, self.Method, self.Path, stage, editor, intrinsics_resolver
             )
+        if self.TimeoutInMillis:
+            self.add_timeout_to_swagger(self.Path, self.Method, self.TimeoutInMillis, editor)
```

Comment:
I see in [this doc](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-integration.html) the `timeoutInMillis` can't be less than 50 and more than 29000. Maybe we can throw an error here if someone puts a num  <50 or >29000

Reply:
If we consider it as a`Passthrough` property, we don't really need any validations here.